js/rpg_managers/ImageManager.js

```diff
@@ -128,3 +136,175 @@ ImageManager.isBigCharacter = function(filename) {
+    this._imageCache.reserve(this._generateCacheKey(path, hue), reservationId);
+
+    return bitmap;
+};
+
+ImageManager.releaseReservation = function(reservationId){
+    this._imageCache.releaseReservation(reservationId);
+};
+
+ImageManager.setDefaultReservationId = function(reservationId){
+    this._defaultReservationId = reservationId;
+};
+
+
+ImageManager.requestAnimation = function(filename, hue) {
+    return this.requestBitmap('img/animations/', filename, hue, true);
+};
+
+ImageManager.requestBattleback1 = function(filename, hue) {
+    return this.requestBitmap('img/battlebacks1/', filename, hue, true);
+};
+
+ImageManager.requestBattleback2 = function(filename, hue) {
+    return this.requestBitmap('img/battlebacks2/', filename, hue, true);
+};
+
+ImageManager.requestEnemy = function(filename, hue) {
+    return this.requestBitmap('img/enemies/', filename, hue, true);
+};
+
+ImageManager.requestCharacter = function(filename, hue) {
+    return this.requestBitmap('img/characters/', filename, hue, false);
+};
+
+ImageManager.requestFace = function(filename, hue) {
+    return this.requestBitmap('img/faces/', filename, hue, true);
+};
+
+ImageManager.requestParallax = function(filename, hue) {
+    return this.requestBitmap('img/parallaxes/', filename, hue, true);
+};
+
+ImageManager.requestPicture = function(filename, hue) {
+    return this.requestBitmap('img/pictures/', filename, hue, true);
+};
+
+ImageManager.requestSvActor = function(filename, hue) {
+    return this.requestBitmap('img/sv_actors/', filename, hue, false);
+};
+
+ImageManager.requestSvEnemy = function(filename, hue) {
+    return this.requestBitmap('img/sv_enemies/', filename, hue, true);
+};
+
+ImageManager.requestSystem = function(filename, hue) {
+    return this.requestBitmap('img/system/', filename, hue, false);
+};
+
+ImageManager.requestTileset = function(filename, hue) {
+    return this.requestBitmap('img/tilesets/', filename, hue, false);
+};
+
+ImageManager.requestTitle1 = function(filename, hue) {
+    return this.requestBitmap('img/titles1/', filename, hue, true);
+};
+
+ImageManager.requestTitle2 = function(filename, hue) {
+    return this.requestBitmap('img/titles2/', filename, hue, true);
+};
+
+ImageManager.requestBitmap = function(folder, filename, hue, smooth) {
+    var path = folder + encodeURIComponent(filename) + '.png';
+    var bitmap = ImageManager.requestNormalBitmap(path, hue);
+    bitmap.smooth = smooth;
+    return bitmap;
+};
+
+ImageManager.requestNormalBitmap = function(path, hue){
+    var key = this._generateCacheKey(path, hue || 0);
+    var bitmap = this._imageCache.get(key);
+    if(!bitmap){
+        bitmap = Bitmap.request(path);
+        bitmap.addLoadListener(function(){
+            bitmap.rotateHue(hue);
+        });
+        this._imageCache.add(key, bitmap);
+        this._requestQueue.enqueue(key, bitmap);
+    }else{
+        this._requestQueue.raisePriority(key);
```

Comment:
I guess the intention of this code is to delete the already requested bitmap from the queue.
Why not implement `removeByKey` instead of `raisePriority`?